src/DynamoCoreWpf/Extensions/ViewStartupParams.cs

```diff
@@ -3,23 +3,27 @@
 using System.Linq;
 using System.Text;
 using Dynamo.Extensions;
+using Dynamo.Core;
+using Dynamo.ViewModels;
+using Dynamo.Models;
 
 namespace Dynamo.Wpf.Extensions
 {
-    public class ViewStartupParams
+    public class ViewStartupParams : StartupParams
     {
+        private readonly DynamoViewModel dynamoViewModel;
+
         /// <summary>
         /// A handle to the extensions that are already constructed in the Model layer
         /// </summary>
         public IExtensionManager extensionManager;
```

Comment:
Let's make this a property `ExtensionManager` and obtain it from the `DynamoModel`.